addon/mirage/utils.js

```diff
@@ -84,8 +102,17 @@ let pipelines = {
 
   audio(file, metadata) {
     let audio = document.createElement('audio');
-    return new RSVP.Promise(function (resolve) {
+    return new RSVP.Promise(function (resolve, reject) {
       audio.addEventListener('loadeddata', resolve);
+      audio.onerror = () => {
+        reject(
+          new Error(
+            'You tried to upload an invalid audio. The upload handler for mirage ' +
```

Comment:
Tiny grammar fix:
I would change "You tried to upload an invalid audio" to "You tried to upload an invalid audio file".

Reply:
:+1: 
Should I change "invalid videos" also to "invalid video files"? How it's about "invalid images"? I'm not a native speaker. Appreciate any help with grammar. :smile: